ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VectorDBTool.java

```diff
@@ -44,8 +44,9 @@ public class VectorDBTool implements Tool {
     public static final String NAME = "VectorDBTool";
     @Setter @Getter
     private String alias;
+    private static String DEFAULT_DESCRIPTION = "Useful for when need to search my data in OpenSearch index.";
```

Comment:
if description does go to the LLM, then probably want to have consistent language.
"Use to search over data in my OpenSearch index."
Does OpenSearch even need to be mentioned here?

Reply:
Good point. Will fine tune all of these later